queries/r/locals.scm

```diff
@@ -1,11 +1,11 @@
 ; locals.scm
 
-(function_definition) @local.scope
+(function_definition) @scope
 
-(formal_parameters (identifier) @local.definition)
+(formal_parameters (identifier) @definition)
 
-(left_assignment name: (identifier) @local.definition)
-(equals_assignment name: (identifier) @local.definition)
-(right_assignment name: (identifier) @local.definition)
+(left_assignment name: (identifier) @definition)
+(equals_assignment name: (identifier) @definition)
+(right_assignment name: (identifier) @definition)
```

Comment:
these are probable `@definition.var`

Reply:
Ok, then it should be fine. One could still argue that all of this are still variables even if you can assign functions to it.